app/facades/word_facade.rb

```diff
@@ -3,8 +3,15 @@ def self.find_word
     response = WordService.word_search
 
     response[:data][:attributes][:word]
-      # response.map do |r|
-      # Word.new(r)
-      # end
+   
+  end
+
+  def self.multi_word_search(num)
+    response = WordService.multi_word(num)
+    words = []
+    
+    response.map do |r|
+      Word.new(r)
```

Comment:
'r' as a variable is vague, is there another way we can label it that is more clear? maybe |res|? 
Make decision as a group?